File: core/src/com/tehreh1uneh/littlesubmarineadventure/SubmarineGame.java
package com.tehreh1uneh.littlesubmarineadventure;

import com.badlogic.gdx.Game;
import com.tehreh1uneh.littlesubmarineadventure.screens.menu_screen.MenuScreen;

public class SubmarineGame extends Game{

	@Override
	public void create () {
		setScreen(new MenuScreen(this));
	}
}

File: core/src/com/tehreh1uneh/littlesubmarineadventure/SubmarineGame.kt
package com.tehreh1uneh.littlesubmarineadventure

import com.badlogic.gdx.Game
import com.tehreh1uneh.littlesubmarineadventure.screens.menuscreen.MenuScreen

class SubmarineGame : Game() {
    override fun create() {
        setScreen(MenuScreen(game = this))
    }
}